matrix/common/etl/transformers/cell_expression.py

```diff
@@ -239,7 +256,9 @@ def _parse_optimus_chunk(self,
                  keys["library_key"],
                  keys["analysis_key"],
                  barcodes[i],
-                 str(gene_count)]
+                 str(gene_count),
+                 str(emptydrops_result[barcodes[i]]["total_umi_count"]),
+                 't' if emptydrops_result[barcodes[i]]["is_cell"] else 'f']
```

Comment:
`emptydrops_is_cell` in table schema is `BOOLEAN`

Reply:
Yeah, the redshift docs say 't' or 'f' are okay literal values for boolean fields. https://docs.aws.amazon.com/redshift/latest/dg/r_Boolean_type.html (I think I'm understanding that right)

Reply:
Oh I see!